### PR TITLE
Contributor Role vocab

### PR DESCRIPTION
**Pull request by vturpin**
Rhange link to the control vocab "https://docs.google.com/document/d/1dN90xkw9oCbLs0sPPhOmszdOjLpwcqxiK5mjeZP7abA/edit#" Following #77 submitted 20th of July 2022

Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [ ] Typo without possible change of interpretation of the related text.
- [ ] Fix of some error, inconsistency, unforeseen limitation.
- [ ] Style that only affects visually the compiled document
- [X] Addition that does not require change in the current structure.
- [ ] Enhancement that require changes to improve the format.

# Related Issues

# Dates when it got review approvals

#77 submitted in July 2022
# Release checklist

- [x] Approved by at least two members of the committee?
- [ ] There were modifications after the review approvals? If so, please
      ask reviewers to update their review.
- [x] Proponents and moderador should explicitly agree that it is ready to
      to merge.
- [x] The moderador is the one in charge to actually merge or close this PR
      according to the final decision.

# For maintainers

- [ ] Update the moderator with a volunteer from the committee. It would be
      best to have one single moderator to guide and help this PR to move
      forward. It is OK to update the moderador pass it to another one.
- [ ] Confirm that the associated branch was deleted after the merging.
- [ ] Wrap-up and close the related issues.

# Comments

### CODE REVIEW

File: OG_Format.adoc [hunk 1]
```diff
@@ -194,7 +194,7 @@ ex.:
 |contributor_email |Email if the contributors to the glider mission. Multiple contributors’ emails are separated by commas. |PI email is mandatory |
 |contributor_id |Unique id of the contributors to the glider mission. Multiple contributors’ ids are separated by commas. |highly desirable |
 |contributor_role |Role of the contributors to the glider mission. Multiple contributors’ roles are separated by commas. |PI vocabulary is mandatory |
-|contributor_role_vocabulary |Controlled vocabulary for the roles used in the "contributors_role". Multiple contributors’ roles and vocabularies are separated by commas. |PI vocabulary is mandatory |https://orcid.org/[_https://orcid.org/_]
```
Comment (castelao) [inline OG_Format.adoc:197]: Oh yeah, ORCID doesn't give role vocabulary. But shouldn't we use orcid for contributor_id?

**Review by castelao — CHANGES_REQUESTED** [review]

File: OG_Format.adoc (hunk 1 above)
Comment (vturpin) [reply OG_Format.adoc:197]: I think orcid can be a exemple of collection. They might be others. I remember that @justinbuck pointed to other collection of contributor's id.

Comment (justinbuck) [reply OG_Format.adoc:197]: Hi @vturpin , try this http://vocab.nerc.ac.uk/search_nvs/W08/
Justin

**Review by castelao — APPROVED** [review]
Make sense to me with the new link that @justinbuck pointed out, but it would be worth someone else double-checking and if it is the case, approve this PR.

File: OG_Format.adoc (hunk 1 above)
Comment (castelao) [reply OG_Format.adoc:197]: A note, we might need to expand the possible options on W08.

**Review by callumrollo — APPROVED** [review]

**Pull request merged**